packages/core/src/form-field/FormField.css

```diff
@@ -26,7 +26,7 @@
   --formField-focused-outlineStyle: var(--uitkFormField-focused-outlineStyle, var(--uitk-focused-outlineStyle));
   --formField-focused-outlineWidth: var(--uitkFormField-focused-outlineWidth, var(--uitk-focused-outlineWidth));
   --formField-helperText-height: 0px;
-  --formField-helperText-lineHeight: var(--uitk-text-base-lineHeight);
+  --formField-helperText-fontSize: var(--uitkFormField-helperText-fontSize, var(--uitk-text-help-fontSize));
```

Comment:
confusing. why are we declaring the helper text vars in here when we have a separate css file for it